packages/rpc-methods/src/restricted/getBip32PublicKey.test.ts

```diff
@@ -79,6 +79,28 @@ describe('getBip32PublicKeyImplementation', () => {
       );
     });
 
+    it('derives the ed25519 public key from the path', async () => {
+      const getUnlockPromise = jest.fn().mockResolvedValue(undefined);
+      const getMnemonic = jest
+        .fn()
+        .mockResolvedValue(TEST_SECRET_RECOVERY_PHRASE_BYTES);
+
+      expect(
+        await getBip32PublicKeyImplementation({
+          getUnlockPromise,
+          getMnemonic,
+          // @ts-expect-error Missing other required properties.
```

Comment:
What other properties? Doesn't this function just take those two params?

Reply:
> TS2345: Argument of type '{ params: { path: ["m", "44'", "1'", "0'", "0'", "1'"]; curve: "ed25519"; }; }' is not assignable to parameter of type 'RestrictedMethodOptions<GetBip32PublicKeyParameters>'.   Type '{ params: { path: ["m", "44'", "1'", "0'", "0'", "1'"]; curve: "ed25519"; }; }' is missing the following properties from type 'RestrictedMethodOptions<GetBip32PublicKeyParameters>': method, context

Reply:
Interesting, why did the error show up here? Shouldn't it have been in the returned function call below?

Reply:
The error happens in the function call (starting from line 93), and the `@ts-expect-error` is just above it (line 92).

Reply:
Oh right 😄